core/connection/session.go

```diff
@@ -25,14 +25,43 @@ import (
 	"github.com/mysteriumnetwork/node/session"
 )
 
+// SessionStatus represents list of possible session statuses
+type SessionStatus int
+
+const (
+	// SessionStatusNew means that newly created session object is written to storage
+	SessionStatusNew = SessionStatus(0)
+	// SessionStatusCompleted means that session object is updated on connection disconnect event
+	SessionStatusCompleted = SessionStatus(1)
+)
+
 // Session holds structure for saving session history
 type Session struct {
 	SessionID       session.ID `storm:"id"`
 	ProviderID      identity.Identity
 	ServiceType     string
 	ProviderCountry string
 	TimeStarted     time.Time
-	// these will be updated while session is ongoing
-	Duration  int // in seconds
-	DataStats stats.SessionStats
+	Status          SessionStatus
+	TimeUpdated     time.Time          // is updated on disconnect event
```

Comment:
If it updated on disconnect event, maybe we should call it `Finished`?

Reply:
Later Session object can be updated during session. So I suggest to leave more generic name